modules/swagger-codegen/src/main/resources/typescript-angular2/api.mustache

```diff
@@ -34,6 +34,21 @@ export class {{classname}} {
             this.configuration = configuration;
         }
     }
+	
+	/**
+     * 
+     * Extends object by coping non-existing properties.
+     * @param objA object to be extended
+     * @param objB source object
+     */
+    private extendObj<T1,T2>(objA: T1, objB: T2) {
```

Comment:
Maybe put this function in a different file and import it in every api? This avoids code duplication in every api.